protocols/ping/src/handler.rs

```diff
@@ -295,8 +295,7 @@ impl ConnectionHandler for Handler {
                         break;
                     }
                     Poll::Ready(Ok((stream, rtt))) => {
-                        log::debug!("latency to {} is {}ms", self.peer, rtt.as_millis());
-
+                        tracing::event!(tracing::Level::TRACE, ?rtt);
```

Comment:
```suggestion
                        tracing::debug!(?rtt, "ping succeeded");
```

I think adding _some_ message here makes sense.